orx-syphon/build.gradle

```diff
@@ -0,0 +1,6 @@
+dependencies {
```

Comment:
I fixed these to be `implementation`, because you don't want these libraries to be exposed in the user's program.